NBXplorer/Controllers/MainController.PSBT.cs

```diff
@@ -312,10 +312,14 @@ public partial class MainController
 				psbt = txBuilder.BuildPSBT(false);
 				hasChange = psbt.Outputs.Any(o => o.ScriptPubKey == change.ScriptPubKey);
 			}
-			catch (OutputTooSmallException)
+			catch (OutputTooSmallException ex) when( ex.Reason == OutputTooSmallException.ErrorType.TooSmallAfterSubstractedFee)
 			{
 				throw new NBXplorerException(new NBXplorerError(400, "output-too-small", "You can't substract fee on this destination, because not enough money was sent to it"));
 			}
+			catch (OutputTooSmallException ex) when( ex.Reason == OutputTooSmallException.ErrorType.TooSmallBeforeSubstractedFee)
+			{
+				throw new NBXplorerException(new NBXplorerError(400, "output-below-dust", "The amount is being sent is below dust threshold"));
+			}
```

Comment:
Why introducing `output-below-dust` ? I believe this is way too much details people already using `output-too-small`, should just use this.